src/domain/entities/Settings.ts

```diff
@@ -0,0 +1,43 @@
+import { Struct } from "$/domain/entities/generic/Struct";
+
+export type AppSettingsAttrs = {
+    attributeGroupId: string;
+    categoryComboId: string;
+    categoryProjectsId: string;
+    createdByDataSetConfigurationAttributeId: string;
+    dataElementGroupGlobalIndicatorMandatoryId: string;
+    dataElementGroupOutputId: string;
+    dataElementGroupSetCoreCompetencyId: string;
+    dataElementGroupSetOriginId: string;
+    dataElementGroupSetStatusId: string;
+    dataElementGroupSetThemeId: string;
+    dataPeriodIntervalDatesAttributeId: string;
+    dataPeriodOutcomeDatesAttributeId: string;
+    dataPeriodOutputDatesAttributeId: string;
+    exclusionRuleCoreUserGroupId: string;
+    expiryDays: number;
+    hideInDataSetAppAttributeId: string;
+    indicatorGroupGlobalIndicatorMandatoryId: string;
+    indicatorGroupSetOriginId: string;
+    indicatorGroupSetStatusId: string;
+    indicatorGroupSetThemeId: string;
+    organisationUnitLevelForCountriesId: string;
+    outcomeEndDate: {
+        day: number;
+        month: number;
+    };
+    outcomeLastYearEndDate: {
+        units: string;
+        value: number;
+    };
+    outputEndDate: {
+        day: number;
+        month: number;
+    };
+    outputLastYearEndDate: {
+        units: string;
+        value: number;
+    };
+};
```

Comment:
note: here we have kind of a mismatch. These are settings of things we consider to be data details (ex: dataElementGroupSets), so conceptually, should not be in the domain. 

But I understand you will want it configurable, so it will have UI so some link with the domain. 

It could be something we pass to the d2 repo in the constructor so it does not appear in the domain, but then the composition root should be recreated after UI changes. And the UI would be referencing some structure not in domain.  So if you have time and that worries you, check with Jorge.

Reply:
I removed this entity since I initially planned to include the settings form in this PR but later realized it would be better to handle it in a separate one

Reply:
Ok. Anyway, you'll face the mismatch, we'll see what's the best approach.